docs/known-issues/known-issues.md

```diff
@@ -0,0 +1,88 @@
+# Known Issues
+
+The following is a list of current limitations of the Fabric, which we are
+working hard to address:
+
+* [Deleting a VPC and creating a new one right away can cause the agent to fail](#deleting-a-vpc-and-creating-a-new-one-right-away-can-cause-the-agent-to-fail)
+* [VPC local peering can cause the agent to fail if subinterfaces are not supported on the switch](#vpc-local-peering-can-cause-the-agent-to-fail-if-subinterfaces-are-not-supported-on-the-switch)
+* [External peering over a connection originating from an MCLAG switch can fail](#external-peering-over-a-connection-originating-from-an-mclag-switch-can-fail)
+* [MCLAG leaf with no surviving spine connection will blackhole traffic](#mclag-leaf-with-no-surviving-spine-connection-will-blackhole-traffic)
+
+### Deleting a VPC and creating a new one right away can cause the agent to fail
+
+The issue is due to limitations in SONiC's gNMI interface. In this particular case,
+the deletion and creation of a VPC back-to-back (i.e. using a script or the golang API)
+can lead to the reuse of the deleted VPC's VNI before the deletion had effect.
+
+#### Diagnosing this issue
+
+The applied generation of the affected agent reported by kubectl will not
+converge to the last desired generation. Additionally, the agent logs on the switch 
+(accessible at `/var/log/agent.log`) will contain an error similar to the following one:
+
+```
+time=2025-03-23T12:26:19.649Z level=ERROR msg=Failed err="failed to run agent: failed to process agent config from k8s: failed to process agent config loaded from k8s: failed to apply actions: GNMI set request failed: gnmi set request failed: rpc error: code = InvalidArgument desc = VNI is already used in VRF VrfVvpc-02"
+```
+
+#### Known workarounds
+
+Deleting the pending VPCs will allow the agent to reconverge. After that, the
+desired VPCs can be safely created.
+
+### VPC local peering can cause the agent to fail if subinterfaces are not supported on the switch
```

Comment:
I _think_ we should say which of our supported switches don't support sub-interfaces. I don't know which ones they are right now. But perhaps that is part of the "limited-leaf" profile we have been discussing?

Reply:
I mention a little further down that users can check whether a switch model supports interfaces by looking at the switch profiles catalogue. Looking at it, there are currently 4 models which don't support subinterfaces:
* Celestica DS4000
* Dell Z9332F-ON
* Edgecore DCS501
* Edgecore EPS203

I'm not necessarily against explicitly mentioning these in the issue, but I'm afraid that with time the list might become obsolete and go out of sync with the information in the catalogue

Reply:
I agree with the concern about the list getting outdated. Perhaps if we further formalize the notion of a "limited-leaf" or similar we can revisit. I think the code is good as is.